setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup
 
-with open("README.rst", "r") as fin:
+with open("README.md", "r") as fin:
```

Comment:
If we decide to stick with `.rst` instead of `.md`, this change will need to be reverted.